arm-insights/2015-04-01/swagger/insightsManagementClient.json

```diff
@@ -2,7 +2,7 @@
   "swagger": "2.0",
   "info": {
     "title": "InsightsManagementClient",
-    "version": "2015-04-01"
+    "version": "2016-03-01"
```

Comment:
The folder name in the path arm-insights/2015-04-01/swagger/insightsManagementClient.json should match the api-version